tests/_data/tests_files/test_integrator_wire_navigation.xml

```diff
@@ -26,6 +26,27 @@
         <bundle>main</bundle>
         <controller>index</controller>
         <action>index</action>
+        <visibility>0</visibility>
+        <pages>
+            <main-nested>
+                <label>Before main - nested</label>
+                <title>Before main - nested</title>
+                <icon>fa-archive</icon>
+                <bundle>before-main-nested</bundle>
+                <controller>index</controller>
+                <action>index</action>
+                <visible>0</visible>
+            </main-nested>
+            <main-nested2>
+                <label>Before main - nested</label>
```

Comment:
Better to add number for the value too `Before main - nested 2`

Reply:
done